app/javascript/modCenter/moderationArticles.jsx

```diff
@@ -6,6 +6,9 @@ export class ModerationArticles extends Component {
     articles: JSON.parse(
       document.getElementById('mod-index-list').dataset.articles,
     ),
+    communityName:
+      document.getElementsByClassName('site-logo')[0]?.firstElementChild
+        .innerText,
```

Comment:
We can't rely on obtaining the community name this way, as most communities use an SVG, and there were will be no `innerText` here to grab. In local development, we have text like 'DEV(local)' by default, which is why it appears to work, but then for others fails (@msarit I'm guessing you have a logo uploaded in your local instance 😄 ).

To be able to get a handle to the community name, we will need to add it as a data attribute in the rails-side, so we can pull it out here. I would suggested adding `data-community-name="<%= community_name %>"` and `id='main-content'` to the `<main>` element in `forem/app/views/moderations/index.html.erb`.

Then here you could grab it via `document.getElementById('main-content').dataset.communityName`. Let me know if you need any help with this!

Reply:
@aitchiss I think we now no longer allow SVG logos but `data-community-name` still seems like the right approach. Do you think we should take over this PR and push it over the finish line with the changes you suggested?

Reply:
@zoeyyandi I hope you don't mind but I've gone ahead and pushed that change to your branch, so we can push these changes over the line 🙂 

In the end, I've placed the data-community-name attribute on the `body` element. It made sense to me to keep this information along with the other community metadata we need, especially since it's likely we'll need to access this community name from other parts of the JS in the future.